source/App.cpp

```diff
@@ -154,15 +160,13 @@ void App::MessageReceived(BMessage *message) {
 }
 
 void App::AboutRequested() {
-	(new BAlert("About InternalMIDI", 
-		"\nInternalMIDI V2.5.3\n\n"
-		"Copyright ©2001 by Werner Freytag\n\n"
-//		"Copyright ©2009/2010 Haiku Inc. \n\n"
-		"InternalMIDI creates a MIDI node for the internal Haiku General MIDI synthesizer.\n"
-		"Use a utility like PatchBay to connect it to your MIDI applications!\n\n"
-		"InternalMIDI is released under MIT license.\n\n"
-		"Project Homepage:\nhttp://dev.osdrawer.net/projects/internalmidi\n", "OK", NULL, NULL, 
-		B_WIDTH_FROM_WIDEST, B_EVEN_SPACING, B_INFO_ALERT))->Go();
+	BAboutWindow* about = new BAboutWindow("InternalMIDI", APP_SIGNATURE);
```

Comment:
```suggestion
	BAboutWindow* about = new BAboutWindow(B_TRANSLATE_SYSTEM_NAME("InternalMIDI"), APP_SIGNATURE);
```

...plus anywhere the app name appears inthe interface,